lib/kuby/commands.rb

```diff
@@ -74,7 +75,7 @@ def self.tasks
           end
         end
 
-        tasks.build(build_args)
+        tasks.build(build_args, args, options[:only])
```

Comment:
I'm concerned about the ambiguity of the name `args` here and elsewhere in the call stack. What would you think about `docker_args` instead?